packages/buttons/stories/5-ChevronButton.stories.tsx

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { Story, Meta } from '@storybook/react';
+import { ChevronButton } from '@zendeskgarden/react-buttons';
+import { Col, Grid, Row } from '@zendeskgarden/react-grid';
+
+export default {
+  title: 'Components/Buttons/ChevronButton',
+  component: ChevronButton
+} as Meta;
+
+export const Default: Story = ({
+  isDanger,
+  size,
+  isPrimary,
+  isBasic,
+  isPill,
+  focusInset,
+  isRotated
+}) => (
+  <Grid>
+    <Row>
+      <Col textAlign="center">
+        <ChevronButton
+          aria-label="Other Options"
+          isDanger={isDanger}
+          size={size}
+          isPrimary={isPrimary}
+          isBasic={isBasic}
+          isPill={isPill}
+          focusInset={focusInset}
+          isRotated={isRotated}
```

Comment:
Is there a way to retain the rotation animation we see under the "Icons" section of https://zendeskgarden.github.io/react-components/buttons/#basic?

Reply:
It looks like Storybook does unmount then mount of the component when a control changes the state causing to animation to not happen. I've not found a way around this yet besides baking in a state change within the example (e.g. have a piece of state in the example control the rotation).

Reply:
bummer. We'll need to come up with a typical way to demonstrate animations, as this is an important aspect of the visual test.

Reply:
We could include a button within the example to help with this?

Reply:
yeah, I think you just helped us arrive at our rule. Animation controls need to be added to the storybook demos in situ.

Reply:
This issue affects transitions on props such as `isPrimary`, `disabled`, etc. It looks like this is an issue with Storybook and there is a work-around: https://github.com/storybookjs/storybook/issues/12255. The transitions should now work as expected.

Reply:
Awesome :tada: